module/VuFindAdmin/src/VuFindAdmin/Controller/ConfigController.php

```diff
@@ -47,7 +47,8 @@ public function homeAction()
     {
         $view = $this->createViewModel();
         $view->setTemplate('admin/config/home');
-        $view->baseConfigPath = \VuFind\Config\Locator::getBaseConfigPath('');
+        $resolver = $this->serviceLocator->get(\VuFind\Config\PathResolver::class);
```

Comment:
The trait is in another module so I'm not sure if that's ok. Also I just replicated the functionality in `enableautoconfigAction`, but the code might be wrong; if local config.ini doesn't exist, it would write to base config.ini.

Reply:
Good point. I view the VuFindAdmin module as having a dependency on the VuFind module, but I suppose it's better to keep things as disentangled as possible.

Good point also regarding enableautoconfigAction; probably would be better to call getLocalConfigPath and fail if it's null, to ensure we don't mess up the core config. Should we make a TODO about that and fix it after the rest of the PR, to keep things simpler here?